libpdbg/kernel.c

```diff
@@ -258,6 +258,28 @@ static int kernel_pib_probe(struct pdbg_target *target)
 	return 0;
 }
 
+struct pdbg_target* get_ody_pib_target(struct pdbg_target *target)
+{
+	uint32_t ocmb_proc = pdbg_target_index(pdbg_target_parent("proc", target));
```

Comment:
This logic is used in multiple places. Can we write a method as follows and invoke it from everywhere?

void pdbg_ody_target_parent_of_class(const char *class_name, struct pdbg_target *target)
{
	uint32_t ocmb_proc = pdbg_target_index(pdbg_target_parent("proc", target));
	uint32_t ocmb_index = pdbg_target_index(target) % 0x8;

	struct pdbg_target *cur_target;
	pdbg_for_each_class_target(class_name, cur_target) {
			uint32_t index = pdbg_target_index(cur_target);
			uint32_t proc = 0;
			if(!pdbg_target_u32_property(cur_target, "proc", &proc)) {
				if(index == ocmb_index && proc == ocmb_proc) {
					target = cur_target;
					break;
				}
			}
		}
	//If it reaches here, that means the target could not so assert NULL?
	assert(NULL);
}